docs/api/en/constants/Renderer.html

```diff
@@ -39,13 +39,15 @@ <h2>Shadow Types</h2>
 		THREE.BasicShadowMap
 		THREE.PCFShadowMap
 		THREE.PCFSoftShadowMap
+		THREE.VSMSoftShadowMap
```

Comment:
Is the name intentional? Shouldn't it be `THREE.VSMShadowMap`? Same with `SHADOWMAP_TYPE_VSM_SOFT` inside the shader, should it be `SHADOWMAP_TYPE_VSM`?

Reply:
No intention behind it, I'd just copy and pasted from one above. Your right `THREE.VSMShadowMap` makes more sense, so it's all updated.